lib/Runtime/Library/JavascriptError.cpp

```diff
@@ -132,6 +85,7 @@ namespace Js
         Var message = args.Info.Count > 1 ? args[1] : scriptContext->GetLibrary()->GetUndefined(); \
         return JavascriptError::NewInstance(function, pError, callInfo, newTarget, message); \
     }
+    NEW_ERROR(Error);
```

Comment:
Using NEW_ERROR here creates a NewErrorInstance method that behaves the same way that all the error subtypes constructors behave - see lines 78-87 for detail